### DISCUSSION TITLE
Dependencies clarification

### DISCUSSION THREAD

**Discussion by alexandra-sh** [christiansandberg/pytest-reporter-html1#14] [Q&A]
Hello @christiansandberg,
could you please clarify why the current lib requires the below libraries?
pytest-rerunfailures
pytest-metadata
pytest-xdist
beautifulsoup4

Thank you in advance

**Comment by christiansandberg** [comment]
It doesn't. It's just the the requirements file for the repo in order to run the tests.

When you install the package with pip they will not be included.
